clearwater-infrastructure/usr/share/clearwater/bin/view-subscriber

```diff
@@ -29,7 +29,7 @@ def main():
     # Check the input is a valid IMPU.
     impu = args.impu
     sip_impu_regex = re.compile('sip:.+@.+$')
-    tel_impu_regex = re.compile('tel:\+?[0-9]+$')
+    tel_impu_regex = re.compile('tel:\+?[0-9]+.*$')
```

Comment:
This matches wildcard tel: URIs, like `tel:+1234!*!` - I've seen that show up in the output of `list-subscribers`, but wasn't able to then view its cached data.